src/SimpleCalculator.java

```diff
@@ -0,0 +1,44 @@
+public class SimpleCalculator {
+        private double firstNumber;
```

Comment:
Usually we use BigDecimal instead of double especially when we dealing with money but it's fine of course for now. You can read more here https://stackoverflow.com/questions/3413448/double-vs-bigdecimal